src/server/pachyderm_test.go

```diff
@@ -2121,6 +2121,42 @@ func TestRecursiveCp(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestPipelineUniqueness(t *testing.T) {
```

Comment:
Yup! Since I changed the behavior above and specified a new error string, I wanted a test for these changes.